core/src/main/java/org/infinispan/statetransfer/InboundTransferTask.java

```diff
@@ -118,7 +118,7 @@ public Address getSource() {
             return completionFuture;
          }
          if (trace) {
-            log.tracef("Requesting segments %s of cache %s from node %s", segmentsCopy, cacheName, source);
+            log.tracef("Requesting state from node %s for segments %s", source, segmentsCopy);
```

Comment:
Sorry I might be missing something, but why take out the cacheName?  Seems we would lose some information in the log messages.  Or is it just information overload? :)